### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by IanHawke**
I hope all of these are obvious trivial typos. I think the only one with any content is removing any references to heading cells from the intro notebook.

### CODE REVIEW

**Comment by Carreau** [comment]
Great Thanks ! I'll let the natives english review that. The US contingent is probably just waking up !

**Comment by minrk** [comment]
Fixes look sensible to me, but it looks like you are still adding fixes.

File: Day-1/Notebook/JavaScript Notebook Extensions.ipynb [hunk 1]
```diff
@@ -34,16 +34,16 @@
...
-    "## Tempering with Notebook app"
+    "## Tampering with Notebook app"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The first tool that is availlable to you and that you shoudl be aware of are browser \"developpers tool\". The exact naming can change across browser, and might require the installation of extensions. But basically they can allow you to inspect/modify the DOM, and interact with the javascript code that run the frontend.\n",
+    "The first tool that is available to you and that you should be aware of is the browser \"developers tool\". The exact naming can change across browser, and might require the installation of extensions. But basically they can allow you to inspect/modify the DOM, and interact with the javascript code that run the frontend.\n",
```
Comment (takluyver) [inline Day-1/Notebook/JavaScript Notebook Extensions.ipynb:44]: developers tool -> developer tools

File: Day-1/Notebook/JavaScript Notebook Extensions.ipynb [hunk 2]
```diff
@@ -34,16 +34,16 @@
...
     "\n",
-    " - In Chrome and safari Developper tools are in the menu [Put mmenu name in english here] \n",
+    " - In Chrome and safari Developer tools are in the menu (More tools, Developer tools)\n",
     " - In firefox you might need to install [Firebug](http://getfirebug.com/)\n",
```
Comment (takluyver) [inline Day-1/Notebook/JavaScript Notebook Extensions.ipynb:47]: Firefox has good developer tools available directly from the main menu now. I know this is unrelated, but may as well fix it now we're looking at this file.

File: Day-1/Notebook/Using nbconvert as a Library.ipynb [hunk 3]
```diff
@@ -568,7 +568,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "@damianavila wrote the Nicholas Plugin to [write blog post as Notebooks](http://www.damian.oquanta.info/posts/one-line-deployment-of-your-site-to-gh-pages.html) and is developping a js-extension to publish notebooks via one click from the web app."
+    "@damianavila wrote the Nicholas Plugin to [write blog post as Notebooks](http://www.damian.oquanta.info/posts/one-line-deployment-of-your-site-to-gh-pages.html) and is developing a js-extension to publish notebooks via one click from the web app."
```
Comment (takluyver) [inline Day-1/Notebook/Using nbconvert as a Library.ipynb:571]: Nicholas -> Nikola (again, while we're looking at this line)

**Comment by takluyver** [comment]
Thanks @IanHawke

**Comment by IanHawke** [comment]
@minrk I've no more fixes to add now, after a second run through.

**Pull request merged**